README.md

```diff
@@ -208,6 +256,50 @@ async function streamExecFile() {
 }
 ```
 
+### streamExecFileWithEvents
+
+**Options:**
+
+These are optional options that can be passed to the `exec` function.
+Neither option is required, and the defaults will reduce the number of calls made to the Model API.
+
+- `cache`: Enable or disable caching.
+- `cacheDir`: Specify the cache directory.
+
+**Usage:**
+
+The script is relative to the callers source directory.
+
+```javascript
+const gptscript = require('@gptscript-ai/gptscript');
+
+const opts = {
+    cache: false,
+};
+
+async function streamExecFileWithEvents() {
+    try {
+        const { stdout, stderr, events, promise } = await gptscript.streamExecFileWithEvents('./test.gpt', "--testin how high is that there mouse?", opts);
+
+        stdout.on('data', data => {
+            console.log(`system: ${data}`);
+        });
+
+        stderr.on('data', data => {
+            console.log(`system: ${data}`);
+        });
+
+		events.on('data', data => {
```

Comment:
Formatting is a bit weird.

Reply:
I think I got the tabs vs spaces figured out here.